### ISSUE TITLE
deprecated codes

### ISSUE THREAD

**Issue by guolingege** [simolus3/sqlite3.dart#21]
```
../../../../.pub-cache/hosted/pub.flutter-io.cn/sqlite3-0.1.9-nullsafety.2/lib/src/ffi/sqlite3.ffi.dart:6:7: Info: Struct 'char' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class char extends Struct {}
      ^
../../../../.pub-cache/hosted/pub.flutter-io.cn/sqlite3-0.1.9-nullsafety.2/lib/src/ffi/sqlite3.ffi.dart:8:7: Info: Struct 'sqlite3' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class sqlite3 extends Struct {}
      ^
../../../../.pub-cache/hosted/pub.flutter-io.cn/sqlite3-0.1.9-nullsafety.2/lib/src/ffi/sqlite3.ffi.dart:10:7: Info: Struct 'sqlite3_stmt' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class sqlite3_stmt extends Struct {}
      ^
../../../../.pub-cache/hosted/pub.flutter-io.cn/sqlite3-0.1.9-nullsafety.2/lib/src/ffi/sqlite3.ffi.dart:12:7: Info: Struct 'sqlite3_value' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class sqlite3_value extends Struct {}
      ^
../../../../.pub-cache/hosted/pub.flutter-io.cn/sqlite3-0.1.9-nullsafety.2/lib/src/ffi/sqlite3.ffi.dart:14:7: Info: Struct 'sqlite3_context' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class sqlite3_context extends Struct {}
      ^
../../../../.pub-cache/hosted/pub.flutter-io.cn/ffi-0.2.0-nullsafety.1/lib/src/utf8.dart:23:7: Info: Struct 'Utf8' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class Utf8 extends Struct {
      ^
../../../../.pub-cache/hosted/pub.flutter-io.cn/ffi-0.2.0-nullsafety.1/lib/src/utf16.dart:16:7: Info: Struct 'Utf16' is empty. Support for empty structs is deprecated and will be removed in the next stable version of Dart. Use Opaque instead.
class Utf16 extends Struct {
      ^
../../../../.pub-cache/hosted/pub.flutter-io.cn/ffi-0.2.0-nullsafety.1/lib/src/allocation.dart:47:33: Info: Support for using non-constant type arguments 'T' in this FFI API is deprecated and will be removed in the next stable version of Dart. Rewrite the code to ensure that type arguments are compile time constants referring to a valid native type.
  final int totalSize = count * sizeOf<T>();
                                ^
```

flutter version:1.27.0
dart version:2.12.0-dev

**Comment by dcharkes** [comment]
Please update the `package:ffi` dependency to 1.0.0 to get rid of the warnings (SDK constraint 2.12.0-beta.9).

**Comment by simolus3** [comment]
The warnings should be fixed if you upgrade to `sqlite3: 0.1.10-nullsafety.0`.
I'll publish a new version with support for the latest ffi release later today.

**Comment by simolus3** [comment]
I've just published version `1.0.0` of `sqlite3` (and version `0.4.0` of `sqlite3_flutter_libs`, but that should be unrelated).
